benchmarks/gpu_pyfixest_errors.ipynb

```diff
@@ -0,0 +1,1478 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    g = sns.FacetGrid(
</pre><p>I think you could use <a href="https://seaborn.pydata.org/generated/seaborn.catplot.html" target="_blank">https://seaborn.pydata.org/generated/seaborn.catplot.html</a> directly </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Done.